doc/development.md

```diff
@@ -28,49 +28,74 @@ If you wish to compile DAOS with clang rather than gcc, set COMPILER=clang on th
 
 ## Go dependencies
 
-Developers contributing Go code may need to change the external dependencies located in the src/control/vendor directory. The DAOS codebase uses [dep](https://github.com/golang/dep) to manage these dependencies.
-
-On EL7 and later:
-
-```
-    yum install yum-plugin-copr
-    yum copr enable hnakamur/golang-dep
-    yum install golang-dep
-```
-
-On Fedora 27 and later:
-
-```
-    dnf install dep
-```
-
-On Ubuntu 18.04 and later:
-
-```
-    apt-get install go-dep
-```
-
-For OSes that don't supply a package:
-* Ensure that you have a personal GOPATH (see "go env GOPATH", referred to as "$GOPATH" in this document) and a GOBIN ($GOPATH/bin) set up and included in your PATH:
-
-```
-    mkdir -p $GOPATH/bin
-    export PATH=$GOPATH/bin:$PATH
-```
-
-* Then follow the [installation instructions on Github](https://github.com/golang/dep).
-
-To update the vendor directory using dep after changing Gopkg.toml, first make sure DAOS is cloned into
-
-```
-    $GOPATH/src/github.com/daos-stack/daos
-```
-
-Then:
-
-```
-    cd $GOPATH/src/github.com/daos-stack/daos/src/control
-    dep ensure
+Developers contributing Go code may need to change the external dependencies located in the src/control/vendor
+directory. The DAOS codebase uses [Go Modules](https://github.com/golang/go/wiki/Modules) to manage these
+dependencies. As this feature is built in to Go distributions starting with version 1.11, no
```

Comment:
below . don't we see that 1.12 is the minimum version?

Reply:
Yeah, because 1.12 is the newest version I can install on OpenSUSE Leap 15. I would make 1.13 the minimum if I could...